Update files-troubleshoot-smb-connectivity.md (#1480)
MicrosoftDocs/SupportArticles-docs: support/azure/azure-storage/files/connectivity/files-troubleshoot-smb-connectivity.md

Revert the `LmCompatibilityLevel` value to the default value of 3 in the following registry subkey:

`HKLM\SYSTEM\CurrentControlSet\Control\Lsa`  

### <a id="error53-67-87"></a> Failed with Error Code: 0x800704b3 the network path was either typed incorrectly, does not exist, or the network provider is not currently available.

#### Cause:

> This error can occur if any core windows network related services are disabled as any services that explicitly depends on it will fail to start.   

Reviewer: ```suggestion
This error can occur if any core Windows network related services are disabled, as any service that explicitly depends on those network services will fail to start.   
```

#### Solutions

> Check if any of the below mentioned services are in **Stopped** state in the Azure Windows VM and if you find any, please start it and retry to mount the Azure file share:  

Reviewer: ```suggestion
Check if any of these services are in a **Stopped** state in the Windows VM. If you find any, start the service(s) and retry mounting the Azure file share.
```
